### PR TITLE
SAP Formulas

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Now with added cluster formulas! I think this probably completes first draft, so once this PR is merged, I would like to build the book (ping @jcayouette ) and send it for technical review. 

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [ ] Manager-4.1
- [x] Manager-4.2
- [x] Uyuni

# Links

Partial https://github.com/SUSE/spacewalk/issues/10772

### CODE REVIEW

File: modules/quickstart-sap/pages/configure.adoc [hunk 1]
```diff
@@ -2,7 +2,7 @@
 = Configure Clients
 
 {productname} uses formulas with forms to configure your SAP clients.
-There are two formulas that you need to use: ``habootstrap`` to configure the HANA database, and ``FIXME`` to configure the clients into a cluster.
+There are two formulas that you need to use: ``Hana`` to configure the HANA database, and ``Cluster`` to configure the clients into a cluster.
```
Comment (keichwa) [inline modules/quickstart-sap/pages/configure.adoc:5]:
```suggestion
There are two formulas that you need to use:

* ``Hana`` to configure the HANA database
* ``Cluster`` to configure the clients into a cluster
```

File: modules/quickstart-sap/pages/configure.adoc [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
 
 The formulas are provided by packages that you can download from your package manager.
```
Comment (keichwa) [inline modules/quickstart-sap/pages/configure.adoc:7]: Drive-by remark: I think you can download the packages _with_ the package manager (I always think about it as a software tool) or _from_ an installation source (repo, etc.). Maybe, I'm blinded by my German language background, though.

File: modules/quickstart-sap/pages/configure.adoc [hunk 3]
```diff
@@ -32,31 +32,29 @@ If you perform these steps in the wrong order, your SAP installation will fail.
 
 In the {productname} {webui}, navigate to menu:Systems[System List] and click the client to use as the primary client in the cluster.
 
-Navigate to the [guimenu]``Formulas`` tab and check the ``habootstrap`` formula in the list.
-Click btn:[Save] and apply the highstate to install the formula.
+Navigate to the [guimenu]``Formulas`` tab, locate the ``Sap Hana Deployment`` heading, and check the ``Hana`` formula in the list.
+Click btn:[Save] and apply the highstate to activate the formula.
 
-When the formula is applied, you can navigate to the menu:Formulas[Habootstrap] tab, and complete the details in the form.
+When the formula is applied, you can navigate to the menu:Formulas[Hana] tab, and complete the details in the form.
```
Comment (keichwa) [inline modules/quickstart-sap/pages/configure.adoc:38]:
```suggestion
When the formula is activated, navigate to the menu:Formulas[Hana] tab, and complete the details in the form.
```

File: modules/quickstart-sap/pages/configure.adoc [hunk 4]
```diff
@@ -32,31 +32,29 @@ If you perform these steps in the wrong order, your SAP installation will fail.
...
 
 Make sure you check [guimenu]``Install required packages`` to install everything you need on the client.
-In the [guimenu]``First node`` field, type the hostname of the client to use as the primary client in the cluster.
+In the [guimenu]``Nodes`` sections, type the hostname of the client to install the HANA database, and provide details for the installation.
 
 Complete the remaining details according to your environment, click btn:``Save``, and apply the highstate.
```
Comment (keichwa) [inline modules/quickstart-sap/pages/configure.adoc:43]:
```suggestion
Complete the remaining details according to your environment, click btn:[Save], and apply the highstate.
```

File: modules/quickstart-sap/pages/configure.adoc [hunk 5]
```diff
@@ -32,31 +32,29 @@ If you perform these steps in the wrong order, your SAP installation will fail.
...
-In the {productname} {webui}, navigate to menu:Systems[System List] and click the ``FIXME`` client.
-This is the FIXME.
-Navigate to the [guimenu]``Formulas`` tab and check the formula in the list.
-Click btn:``Save`` and apply the highstate to save the changes.
+In the {productname} {webui}, navigate to menu:Systems[System List] and click the client to use as the primary client in the cluster.
+
+Navigate to the [guimenu]``Formulas`` tab, locate the ``Cluster`` heading, and check the ``Cluster`` formula in the list.
+Click btn:[Save] and apply the highstate to activate the formula.
 
 When the formula is applied, you can navigate to the menu:Formulas[Cluster] tab, and complete the details in the form.
```
Comment (keichwa) [inline modules/quickstart-sap/pages/configure.adoc:55]:
```suggestion
When the formula is activated, navigate to the menu:Formulas[Cluster] tab, and complete the details in the form.
```

File: modules/quickstart-sap/pages/configure.adoc [hunk 6]
```diff
@@ -32,31 +32,29 @@ If you perform these steps in the wrong order, your SAP installation will fail.
...
-//Make sure you ... FIXME
 
-Make sure that you provide the passwords for all the other clients you want included in the cluster.
-Alternatively, you need to ensure that all of the clients can connect to each other directly using SSH.
+Make sure you check [guimenu]``Install required packages`` to install everything you need on the client.
+Give your cluster a name, and specify the hostname of the primary client in the cluster.
 
-When you have completed all the details, click btn:``Save`` and apply the highstate.
-When the highstate is complete, your SAP cluster is ready to use.
+Complete the remaining details according to your environment, click btn:``Save``, and apply the highstate.
```
Comment (keichwa) [inline modules/quickstart-sap/pages/configure.adoc:60]:
```suggestion
Complete the remaining details according to your environment, click btn:[Save], and apply the highstate.
```

**Review by keichwa — APPROVED** [review]

File: modules/quickstart-sap/pages/configure.adoc (hunk 2 above)
Comment (Loquacity) [reply modules/quickstart-sap/pages/configure.adoc:7]: No, I think that's pretty reasonable, actually. I'll update.

File: modules/quickstart-sap/pages/configure.adoc (hunk 4 above)
Comment (Loquacity) [reply modules/quickstart-sap/pages/configure.adoc:43]: :woman_facepalming:

**Review by keichwa — APPROVED** [review]

**Review by jcayouette — APPROVED** [review]
